### ISSUE TITLE
Ambiguity on Apps coverage by policy

### ISSUE THREAD

**Issue by ianwright-msft** [MicrosoftDocs/microsoft-365-docs#5153]
The page is not clear on whether Safe Links protection for O365 apps applies to all users or just licensed users or just users who are in active policies.  We have conflicting statements:

"Safe Links protection for Office 365 apps is applied to all users in the organization who are licensed for Defender for Office 365, regardless of whether the users are included in active Safe Links policies or not."

"Global Safe Links settings: These settings are configured globally, not in Safe Links policies. But, the settings apply only to users who are included in active Safe Links policies. These settings include: Safe Links settings for Office 365 apps"

And then in the situations that explain if a user is covered or not, it indicates the user _does_ need to be in an active policy to get coverage for O365 apps.  If this is now indeed the case, can we get the line updated or removed that says O365 apps are covered for licensed users regardless of whether they're in an active policy?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8112b7a9-8f21-18b3-03da-f8c02b05f239
* Version Independent ID: a441f1d1-5844-ecb9-ba44-95b94714e4e8
* Content: [Safe Links - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/safe-links?view=o365-worldwide#block-the-following-urls-list-for-safe-links)
* Content Source: [microsoft-365/security/office-365-security/safe-links.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/safe-links.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@ianwright-msft, the update for stating that Safe Links for Office aps works for all licensed users (not just members of Safe Links polices) was added recently.

I thought I scrubbed the topic for conflicting references, but it looks like I missed one. That offending statement now says:

> **Global Safe Links settings**: These settings are configured globally, not in Safe Links policies. These settings include:

Basically, what happening is Safe Links for Office 365 apps is all licensed users. The "Block the following URLs" list for Safe Links applies to both email (which requires membership in a Safe Links policy) and Office 365 apps (which does not).
